css-align-3/resources/alignment-parsing-utils.js

```diff
@@ -0,0 +1,19 @@
+var position = [ "start", "end", "left", "right", "center", "flex-start", "flex-end"];
+var distribution = [ "stretch", "space-around", "space-between", "space-evenly"];
+var baseline = [ "baseline", "last-baseline"];
```

Comment:
I can agree on the proposal for a different name, but these should be expected to be used just in alignment related tests, so adding "Alignment" looks like redundant.